src/main/java/impl/storage/Storage.java

```diff
@@ -0,0 +1,51 @@
+package impl.storage;
+
+import impl.chatbot.Danny;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Objects;
+import java.util.Scanner;
```

Comment:
I think you need to put all the import java.* statement above all the import other.* statement